packages/qdrant-loader/src/qdrant_loader/cli/commands/webhook_cmd.py

```
@@ -68,4 +72,5 @@ async def run_webhook_command(
         await server.serve()
     except Exception as exc:
         logger.exception("Webhook server error")
-        raise ClickException(f"Webhook server failed: {exc}") from exc
+        safe_error = sanitize_exception_message(exc) or type(exc).__name__
+        raise ClickException(f"Webhook server failed: {safe_error}") from exc
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)